awx_collection/plugins/lookup/controller_api.py

```diff
@@ -12,7 +12,7 @@
   - None
 description:
   - Returns GET requests from the Automation Platform Controller API. See
-    U(https://docs.ansible.com/ansible-tower/latest/html/towerapi/index.html) for API usage.
+    U(https://docs.ansible.com/automation-controller/latest/html/towerapi/) for API usage.
```

Comment:
@tvo318 and @gundalow Maybe I missed some of the convo here about the `awx_collection` but I feel like it would be much better to avoid sending folks to `docs.ansible.com` for any platform / controller content. In fact the linked `docs.ansible.com` url here basically tells you to go to `redhat.com` so we might as well give them a more direct url. Or am I missing something?

https://developers.redhat.com/api-catalog/api/ansible-automation-controller

Reply:
https://docs.ansible.com/automation-controller/latest/html/towerapi/ has two links:

> The Automation Controller API Guide for Red Hat Ansible Automation Platform has been moved to the [Red Hat Developer page](https://developers.redhat.com/api-catalog/api/ansible-automation-controller).
> For the latest community documentation on Read The Docs, refer to [AWX API Reference](https://ansible.readthedocs.io/projects/awx/en/latest/rest_api/index.html).

Reply:
@oraNod - I discussed this with @gundalow on our 1:1 and we agree just to merge what we have. The decision to use this link is just a temporary fix. I chose the link on docs.ansible because it gives readers a choice on where to go. We can fix this later with another PR when we have a better solution.